### PR TITLE
Fixes evidence bags showing ghost contents after their contents get deleted

### LINKED ISSUE #9544: If you put a primed flashbang (any other grenade?) inside an evidence bag, it "stays" there after detonation.
Basically, prime a flashbang, put it inside the evidence bag. After it detonates, grenade is still in the bag, in a primed state (blinking lights). Trying to empty the bag doesn't get rid of it, gotta put something inside and then empty it to get rid of flashbang icon.

Minor thing, yet still a bug, so reporting it here.

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
Fixes #9544

On a related note, I cannot reproduce #20795 but it might've been caused by this bug leaving a ghost in the evidence bag that was used to bag the item.

[Changelogs]: 
[]: 

:cl: Naksu
fix: Evidence bags will no longer show ghosts of items such as primed flashbangs after the flashbang inside explodes
/:cl:

[why]: 
bugfix

### CODE REVIEW

File: code/modules/detectivework/evidence.dm [hunk 1]
```diff
@@ -17,6 +17,12 @@
 	if(evidencebagEquip(I, user))
 		return 1
 
+/obj/item/evidencebag/handle_atom_del(atom/A)
+	cut_overlays()
+	w_class = WEIGHT_CLASS_TINY
+	icon_state = "evidenceobj"
+	desc = "An empty evidence bag."
```
Comment (Cyberboss) [inline code/modules/detectivework/evidence.dm]: initial for icon_state and desc

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/detectivework/evidence.dm (hunk 1 above)
Comment (Cyberboss) [reply code/modules/detectivework/evidence.dm]: and w_class

Comment (vuonojenmustaturska) [reply code/modules/detectivework/evidence.dm]: eh, might as well

**Comment by vuonojenmustaturska** [comment]
tested and still works

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**